src/components/common/Tabs/Tabs.module.css

```
@@ -0,0 +1,41 @@
+.tab-list {
+  /* [tabListStyles.base] n/a */
+  --tablist-gap-x: 0;
+  --tablist-gap-y: 0;
+  /* [tabListStyles.base] p-1 */
+  --tablist-padding-x: 0.25rem;
+  --tablist-padding-y: 0.25rem;
+  /* [tabProps.base] px-4 py-2 */
+  --tab-padding-x: 1rem;
+  --tab-padding-y: 0.5rem;
+
+  --count: 1; /* updated in Tabs.tsx > TabList */
+  --active: 0; /* updated in Tabs.tsx > TabList  */
+}
+
+.tab-list::after {
+  /* Tailwind's `after:transition` class is not working for me :/ */
+  transition: translate 0.2s ease;
```

```suggestion
  transition: translate 0.2s ease-out;
```
Personal preference: `ease-out` is nice for transitions since it's fast at the start (reacts quickly) and slow at the end (gradually finds its place)

---

Smart!